db/migrate/20180704165549_add_colum_references_to_tickets.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumReferencesToTickets < ActiveRecord::Migration[5.1]
```

Comment:
Amigo falta hacer otra migración que incluya algo como esto:
`add_foreign_key :private_messages, :users, column: :sender_id, primary_key: :id
 add_foreign_key :private_messages, :users, column: :recipient_id, primary_key: :id`

 [mirá acá por ejemplo](https://stackoverflow.com/questions/2057210/ruby-on-rails-reference-the-same-model-twice)

Reply:
You sure?? pa eso no es esto:

  has_many :entry_tickets, class_name: 'Ticket', foreign_key: 'entry_user_id'
  has_many :exit_tickets, class_name: 'Ticket', foreign_key: 'exit_user_id'

Reply:
Esa esa la cosa ahí les estás diciendo que el foreign_key es por ejemplo "entry_user_id"
Pero si vas al schema y ves los foreing keys (que están siempre al final) ese no está definido entonces aunque está guardando los ids como integers en la base de datos, active record probablemente va a fallar en algún momento...

Reply:
Como lo solucionamos then?